packages/jest-mock/src/index.js

```diff
@@ -669,6 +670,33 @@ class ModuleMockerClass {
     this._spyState.forEach(restore => restore());
     this._spyState = new Set();
   }
+
+  stub(object: Object, propertyName: string, mockImplementation?: Function) {
+    if (!object) {
+      throw new Error('object argument must be present');
+    }
+
+    if (!propertyName || typeof propertyName !== 'string') {
+      throw new Error(
+        'propertyName argument must be present and must be a string',
+      );
+    }
+
+    object[propertyName] = this.fn(mockImplementation);
```

Comment:
It should be an error to stub a `propertyName` if it isn't already defined on `object.`